modules/Cargo.toml

```diff
@@ -51,6 +51,7 @@ thiserror = { version = "1.0" }
 protobuf = { version = "2", features = ["with-bytes"] }
 
 clap = { version = "4.0.32", features = ["derive"] }
+chrono = "0.4" 
```

Comment:
I think this could potentially be a big dependency to add. Do you have a rough idea on how much it impacts wasm size? @Buckram123 I remember you saying something about this as well.

Reply:
Yeah, it's pretty big with default features. I've had some issue with it, since it was exposing non default cosmwasm methods, when `default-features=true`
Decided to just check sizes (with optimizer) this contract, results:
- 502K with chrono
- 362K with chrono (default-features = false)
- 359K without chrono

So I think just disabling default features would be good enough!